site/_docs/reference.md

```diff
@@ -2850,6 +2850,7 @@ In the following:
 | b | REGEXP_SUBSTR(string, regexp [, position [, occurrence]]) | Synonym for REGEXP_EXTRACT
 | b m p s | REPEAT(string, integer)                  | Returns a string consisting of *string* repeated of *integer* times; returns an empty string if *integer* is less than 1
 | b m | REVERSE(string)                              | Returns *string* with the order of the characters reversed
+| s | REVERSE(string | array)                        | Returns *string* with the characters in reverse order or array with elements in reverse order
```

Comment:
Because has support the `SELECT REVERSE(5); `So need to update the doc REVERSE(numeric) ?

Reply:
If this is an implicit cast to string, no.

Reply:
Unnecessary, types supported  through implicit conversion, not specified separately by default. 
For example, the spark function document does not specifically explain.

Reply:
Sorry, I don't understand why implicit conversion don't need to log the doc. The doc pay close attention to how to use the func, If this func support , Then we  should illustrate it.

Reply:
I do not think so, you can arbitrarily find a func with a string as args, and funcName(integer) is supported.
<img width="273" alt="image" src="https://github.com/apache/calcite/assets/7956306/e45a312f-401c-400b-abef-2e5a7716dab3">

Implicit conversions are supported because `FamilyOperandTypeChecker#checkOperandTypes` is handled:
<img width="663" alt="image" src="https://github.com/apache/calcite/assets/7956306/9ffe07d3-ff4b-4e24-8c52-459b3f04530f">

More references: [Design of Calcite Implicit Type Cast](https://docs.google.com/document/d/1g2RUnLXyp_LjUlO-wbblKuP5hqEu3a_2Mt2k4dh6RwU/edit#heading=h.bnu0wb9cokyk)

Reply:
+1. Thanks for the explaination.